roles/telemetry_autoscaling/tasks/creating_stack.yml

```diff
@@ -71,15 +71,15 @@
   ansible.builtin.debug:
     var: result
 
-- name: Verify that an instance was launched by the stack creation
+- name: RHOSO-12651 Verify that an instance was launched by the stack creation
   ansible.builtin.shell: |
     # source ~/overcloudrc;
     export STACK_ID=$({{ openstack_cmd }} stack show {{ stack_name }} -c id -f value);
     {{ openstack_cmd }} server list --long | grep $STACK_ID;
   register: result
   failed_when: result.rc >= 1
 
-- name: Verify that the alarms were created for the stack
+- name: RHOSO-12652 Verify that the alarms were created for the stack
```

Comment:
Technically, this would pass if the stack creation didn't work, but there were already alarms in the environment.

So this check needs to be updated.
(not in the scope of this change)

Reply:
I think that a fail condition should be added, for `$alarm_name not in result.stdout`